demisto_sdk/commands/format/update_layout.py

```diff
@@ -238,8 +238,8 @@ def remove_copy_and_dev_suffixes_from_layoutscontainer(self):
                 container = self.data.get(kind)
                 break
         if container:
-            for tab in container.get('tabs', ()):
-                for section in tab.get('sections', ()):
+            for tab in container.get('tabs', ()) or ():
+                for section in tab.get('sections', ()) or ():
```

Comment:
```suggestion
            for tab in (container.get('tabs') or ()):
                for section in (tab.get('sections') or ()):
```
removed redundancy, improved clarity